be/tool/beck.c

```diff
@@ -748,14 +847,21 @@ static int recdo(struct scanner *s, const struct m0_format_tag *tag,
 	result = get(s, buf, size);
 	if (result == 0) {
 		if (memcmp(tag, &r->r_tag, sizeof *tag) == 0) {
+			/*
+			 * Check generation identifier before format footer
+			 * verification. Only process the records from most
+			 * recent generation identifier.
```

Comment:
[* doc *] "most recent generation identifier" - this is not what I see inside `ro_check()` implementations.

Reply:
modified comment.